packages/nextjs/app/api/grants/submit/route.ts

```diff
@@ -0,0 +1,59 @@
+import { NextResponse } from "next/server";
+import { recoverTypedDataAddress } from "viem";
+import { getGrantById, submitGrantBuild } from "~~/services/database/grants";
+import { EIP_712_DOMAIN, EIP_712_TYPES__SUBMIT_GRANT } from "~~/utils/eip712";
+import { PROPOSAL_STATUS } from "~~/utils/grants";
+
+type ReqBody = {
+  grantId?: string;
+  signer?: string;
+  link?: string;
+  signature?: `0x${string}`;
+};
+
+// TODO: Maybe we could fetch submmited build on BG app from firebase and check if its really present,
+// also check that build was actually submitted by the same builder
```

Comment:
cc @carletex, should we add this check too or lol its too much ?

Reply:
haha, I think it's not worth it.